src/XRP/model/xrp-signer.ts

```diff
@@ -15,10 +16,28 @@ export default class XrpSigner {
    */
   public static from(signer: Signer): XrpSigner | undefined {
```

Comment:
I think we can maybe get rid of `| undefined` here too?

Reply:
Oops, missed that